apps/hannk/util/model_runner.cpp

```diff
@@ -571,7 +571,7 @@ int ModelRunner::parse_flags(int argc, char **argv, std::vector<std::string> &fi
 }
 
 void ModelRunner::run(const std::string &filename) {
-    std::cout << "Processing " << filename << " ...\n";
+    std::cout << "\nProcessing " << filename << " ...\n";
```

Comment:
Can we not do this? Having one line per output is *dramatically* better than 2 lines per output when running a lot of stuff (100 lines is a lot better than 200 lines). And it means we can copy/paste into spreadsheets without it being a pain.

Reply:
FWIW, running with --benchmark will always output multiple lines per, and this change makes it IMHO much easier to read, but I'll back it out